core/jobs_test.py

```diff
@@ -435,41 +435,41 @@ def test_cancelling_one_unfinished_job(self):
     def test_compress_output_list_with_single_char_outputs(self):
         input_list = [1, 2, 3, 4, 5]
         expected_output = ['1', '2', '3', '<TRUNCATED>']
-        actual_output = jobs.BaseJobManager._compress_output_list(  # pylint: disable=protected-access
+        actual_output = jobs.BaseJobManager.compress_output_list(  # pylint: disable=protected-access
```

Comment:
Ditto here and elsewhere you've made the change.

Reply:
Hi Kevin,

Thank you for your review. I will quickly modify the code. Do I need to make another pull request?

Reply:
no need, just push your changes to this branch.